docs/en-US/component/dialog.md

```diff
@@ -154,6 +154,9 @@ dialog/events
 | top                        | value for `margin-top` of Dialog CSS, default is 15vh                                                | ^[string]                           | ''      |
 | modal                      | whether a mask is displayed                                                                          | ^[boolean]                          | true    |
 | modal-class                | custom class names for mask                                                                          | ^[string]                           | —       |
+| header-class ^(2.9.2)      | custom class names for header slot                                                                   | ^[string]                           | —       |
+| body-class ^(2.9.2)        | custom class names for default slot                                                                  | ^[string]                           | —       |
+| footer-class ^(2.9.2)      | custom class names for footer slot                                                                   | ^[string]                           | —       |
```

Comment:
I feel that the description here may lead to misunderstandings, as these attributes actually apply to the wrapper of the slot